LogicMonitor.DataSDK/Model/LogsV1.cs

```diff
@@ -23,15 +24,18 @@ public LogsV1()
         {
         }
 
-        public LogsV1(string message, Dictionary<string, string> resourceIds, string timeStamp = default, Dictionary<string, string> metaData = default)
+        public LogsV1(string message, Dictionary<string, string> resourceIds, string timeStamp = default, Dictionary<string, string> metaData = default, LogLevel loglevel=default)
```

Comment:
Setting loglevel to default assign "Trace" as loglevel . Whenever loglevel is not assigned it is expected to be empty string.

Reply:
default end up being an empty string; this has nothing to do with Traces.